install.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
```

Comment:
needs to be called `configure.sh` not `install.sh`, it configures the requirements for building the code.

or

`install_requirements.h` not `install.sh`